tutorials/workflow/csharp/external-system-interaction/ExternalEvents/Program.cs

```diff
@@ -0,0 +1,29 @@
+using Dapr.Workflow;
+using ExternalEvents;
+using ExternalEvents.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDaprWorkflow(options => {
+    options.RegisterWorkflow<ExternalEventsWorkflow>();
+    options.RegisterActivity<RequestApproval>();
+    options.RegisterActivity<ProcessOrder>();
+    options.RegisterActivity<SendNotification>();
+});
+var app = builder.Build();
+
+app.MapPost("/start", async (Order order) => {
+    Console.WriteLine($"Received order: {order}.");
+    await using var scope  = app.Services.CreateAsyncScope();
```

Comment:
Nope, as in a previous message- this isn't necessary. Inject `DaprWorkflowClient` into the async method (can use an optional `[FromServices]` attribute if you want) and access the instance from there. A separate scope is created by invocation of the inbound post request itself and isn't needed here.